### PR TITLE
Add/standard formatter

### PR DESCRIPTION
**Pull request by digitalsadhu**

### CODE REVIEW

**Comment by anotheredward** [comment]
This PR reminds me that this repo really needs some basic tests
LGTM :+1:

File: package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "description": "Wrapper for ActiveDirectory that gets the group for the username passed in through the x-iisnode-auth_user header",
   "main": "index.js",
   "scripts": {
+    "lint": "standard --verbose | snazzy",
```
Comment (ebuckley) [inline package.json:7]: Could avoid the need for global install of standard/snazzy by changing the lint script like this.

``` json
{
    "lint": "./node_modules/.bin/standard --verbose | ./node_modules/.bin/snazzy"
}
```

**Comment by ebuckley** [comment]
LGTM :+1:

File: package.json (hunk 1 above)
Comment (digitalsadhu) [inline package.json:7]: It's already not global. npm is kind enough to use local if available in npm scripts so no need to put in long ./node_modules/.bin type paths.

Comment (ebuckley) [inline package.json:7]: Cool! today I learned NPM is smarter than I give it credit for.

**Pull request merged**